internal/clients/stubs/azure_stub.go

```diff
@@ -2,16 +2,19 @@ package stubs
 
 import (
 	"context"
+	"errors"
+	"fmt"
 	"strconv"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute"
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources"
 	"github.com/RHEnVision/provisioning-backend/internal/clients"
-	"github.com/RHEnVision/provisioning-backend/internal/models"
-	"github.com/RHEnVision/provisioning-backend/internal/ptr"
 )
 
+var NotStartedVMError = errors.New("the VM under given resumeToken not started")
```

Comment:
```suggestion
var ErrNotStartedVM = errors.New("the VM under given resumeToken not started")
```

Prefix Err for values, Error suffix for types. We must enable linter once we fix them all :-)